sources/procfs.go

```diff
@@ -675,6 +676,34 @@ func parseFileElements(path string, directoryDepth int) (name string, nodeName s
 	return name, nodeName, nil
 }
 
+func convertToBytes(s string) string {
+	if len(s) < 1 {
+		return s
+	}
+	new_s := ""
+	multiplier := float64(1)
+	switch finalChar := strings.ToUpper(s[len(s)-1:]); finalChar {
```

Comment:
They are currently uppercase in every situation that we've seen. But...we've been down this road before with string parsing in another project that I help maintain. Any time you can eliminate case sensitivity bugs, you sleep just a little better. :)